README.md

```diff
@@ -139,6 +147,19 @@ Generates an Etch sign URL for an Etch Packet signer. The Etch Packet and its si
     * `clientUserId` (String) - your user eid
     * `signerEid` (String) - the eid of the Etch Packet signer, found in the response of the `createEtchPacket` instance method
 
+##### downloadDocuments(documentGroupEid[, options])
+
+Returns the PassThrough stream of the document group specified by the documentGroupEid in Zip file format.
+* `documentGroupEid` (string) - the eid of the document group to download
+* `options` (Object) - _optional_ Any additional options for the request
+  * `dataType` (Enum[String]) - _optional_ Set the type of the `data` value that is returned in the resolved `Promise`. Defaults to `'buffer'`, but `'stream'` is also supported.
+* Returns a `Promise` that resolves to an `Object`
+   * `statusCode` (Number) - the HTTP status code, `200` is success
+   * `response` (Object) - the Response object resulting from the client's request to the Anvil app
+   * `data` (object) - a PassThrough Stream object containing the document group's data in Zip file format
```

Comment:
Should update this to be more in-line with what's now actually going to happen: buffer of stream. Probably worth following the pattern of `fillPDF` and/or updating both.

Reply:
Will update!